languages/python/reference/exercise-concepts/variable-length-quantity.md

```diff
@@ -0,0 +1,17 @@
+# Concepts of variable-length-quantity
+
+[Example implementation](https://github.com/exercism/python/blob/master/exercises/variable-length-quantity/example.py)
+
+
+## General
+- Loops: concept required to solve the exercise, `while`, `for`
+- Number comparison: concept required to solve the exercise
+- Number sum: concept required to solve the exercise
```

Comment:
Since `sum` is a builtin like `len` it should probably be formatted the same way.